cmd/goplay/main.go

```diff
@@ -51,11 +54,18 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	cwd = dir
+
 	defer os.RemoveAll(dir)
 	file := path.Join(dir, "a.go")
 	if err := ioutil.WriteFile(file, []byte(HelloProg), 0600); err != nil {
 		log.Fatal(err)
 	}
+	modInitCmd := exec.Command("go", "mod", "init", "foo.bar/goplay")
+	modInitCmd.Dir = dir
```

Comment:
Just call `os.Chdir` so you don't need this.

Reply:
👍